### PR TITLE
Adds new integration [luuuis/hass_wibeee]

### PR DESCRIPTION
**Pull request by luuuis**
Originally added in https://github.com/hacs/default/pull/627, the abacao integration is not under active development any more whereas my fork has moved on significantly with improved reliability, multiple device support, energy support, a config flow for UI configuration and just keeping up with new HA versions in general ([see CHANGELOG](https://github.com/luuuis/hass_wibeee/blob/729f61b8bc3a4204851e17a36f1f60f10ba03cee/CHANGELOG.md)).

It uses semver, has tagged/zipped releases and I am planning further improvements such as Local Push support.

### CODE REVIEW

File: integration [hunk 1]
```diff
@@ -4,7 +4,6 @@
   "9a4gl/hass-peltec",
   "9rpp/securifi",
   "AaronDavidSchneider/SonosAlarm",
-  "abacao/hass_wibeee",
```
Comment (ludeeus) [inline integration:7]: This PR should not remove.
Removal needs to happen like this: <https://hacs.xyz/docs/publish/remove>

**Review by ludeeus — CHANGES_REQUESTED** [review]

File: integration (hunk 1 above)
Comment (luuuis) [reply integration]: Sure thing, I will kick off a separate PR for the removal.

Is there a recommended way to handle transitioning users from one integration to another? I was thinking both integrations could coexist in HACS default for a while and in the meantime I can work with @abacao to maybe do a new release just to be able to inform users of the upcoming changes in the release notes. Unless there's a better way?

Comment (abacao) [reply integration]: Let me know when there is a PR to remove my version

Comment (ludeeus) [reply integration]: Remove when you want (assuming old still works)

Comment (luuuis) [reply integration]: Removal PR is https://github.com/hacs/default/pull/1242

**Pull request merged**